cluster/examples/kubernetes/ceph/cluster-on-local-pvc.yaml

```diff
@@ -231,6 +231,8 @@ spec:
               volumeMode: Block
               accessModes:
                 - ReadWriteOnce
+    # when onlyApplySpecificOSDPlacement false, will merge both placement.All() and storageClassDeviceSets.Placement
+    onlyApplySpecificOSDPlacement: false
```

Comment:
should we add  this field in osd on disks example CRs as well?

Reply:
added.